src/PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -465,15 +468,16 @@ private void Init()
             try
             {
                 // If already initialized, need not be on the UI thread
+                // Note that _initialized is only set to true once per VS instance
                 if (!_initialized)
                 {
-                    _initialized = true;
```

Comment:
The problem is that we don't listen to solution saved event. I don't think there is one: https://msdn.microsoft.com/en-us/library/aa301357(v=vs.71).aspx. So, once, _initialized is set to true, saving the solution does not reset the cache

As I say this, I realize that it is possible to get into this situation, even after _initialized is set to true. In other words, after VS NuGet Package has loaded and _initialized has been to true once on VSSolutionManager, one could create a new project or open a project without a solution, and we will get into the same situation. So, this change is incorrect

Perhaps, we can ask the prompt the user to re-open solution after saving the solution